crosshatch/__init__.py

```diff
@@ -92,10 +93,11 @@ def run(self):
                 break
             else:
                 if command == "lyxal":
-                    self.console.print("we do a little trolling")
-                    import webbrowser
-                    webbrowser.open("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
-                    continue
+                    try:
+                        os.system(r"curl -s -L https://raw.githubusercontent.com/keroserene/rickrollrc/master/roll.sh | bash")
+                    finally:
+                        self.console.clear()
+                        break
```

Comment:
help

```suggestion
```

Reply:
Already reverted in 341349da598725fd0626b0e8040c4d932c76a526 :p

Reply:
i wasn't serious lol you can use the other rickroll

Reply:
@Steffan153 It is, to say the least, not a good idea to curl random scripts from the internet